policies/aws_cloudtrail_policies/aws_cloudtrail_s3_bucket_public.py

```diff
@@ -1,11 +1,16 @@
 from panther_base_helpers import deep_get
+from panther_default import aws_regions
 from panther_oss_helpers import BadLookup, resource_lookup
 
 BAD_PERMISSIONS = {
     "http://acs.amazonaws.com/groups/global/AuthenticatedUsers",
     "http://acs.amazonaws.com/groups/global/AllUsers",
 }
 
```

Comment:
Let's add a comment with the AWS doc https://docs.aws.amazon.com/codepipeline/latest/userguide/reference-ct-placeholder-buckets.html

Reply:
Added in https://github.com/panther-labs/panther-analysis/pull/951/commits/8aa67578508d506564e6cc9db8413bf46488a974.